src/mpcCoreKit.ts

```diff
@@ -106,6 +110,22 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
 
     this._tssLib = options.tssLib;
     this._keyType = options.tssLib.keyType as KeyType;
+    this._sigType = (() => {
+      if (!(options.tssLib as V4TSSLibType).sigType) {
```

Comment:
is this required to make sure that mpc core kit works with older tss libs which doesn't have sigType ?

Reply:
yep, basically, if we don't have the sigtype from the lib, then we derive it from the key type.

this is only needed if we want to maintain backwards compatibility. if don't need this, it's more straightforward.

you said we probably want backwards compatibility, but you didn't give a reason yet.

so, i think if we don't really need backwards compatibility, it would be better to drop it because it simplifies the code and makes it less error prone.